app/views/front/dashboard.html.erb

```diff
@@ -46,36 +46,42 @@
             <tbody>
               <% @softwarerecords_indesign.each do |record_indesign| %>
                 <tr>
-                  <td><%= record_indesign.title %></td>
+                  <td>
+                    <% if record_indesign.url.empty? %>
+                      <%= record_indesign.title %>
+                    <% else %>
+                      <a href=<%= generate_path(record_indesign.url) %> target="_blank"><%= record_indesign.title %></a>
```

Comment:
I think we may still want this to link to the View Page. 
We want to remove the tentative installed date and replace it with url and then make the url linkable.

For the production module, you will need to add it after Current Version and before the CRUD icons.

Reply:
I think we have "View" icon which does the same. (link to the view page). Having URL alone would suffice i guess. Need you inputs.